src/net/if_.rs

```diff
@@ -17,3 +17,40 @@ pub fn if_nametoindex<P: ?Sized + NixPath>(name: &P) -> Result<c_uint> {
         Ok(if_index)
     }
 }
+
+libc_bitflags!(
+    /// Standard interface flags, used by `getifaddrs`
+    pub flags InterfaceFlags: libc::c_int {
+        #[cfg(any(target_os = "linux", target_os = "android"))]
```

Comment:
The `target_os` list should be alphabetized.

Reply:
libc defines most of these constants for most platforms, not just Linux.  You should expand the `#[cfg()]` directives appropriately.